src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const daylyRent = 40;
```

Comment:
Consider using a constant for the daily rent value to avoid magic numbers. For example, `const DAILY_RENT = 40;`.